calibrate.py

```diff
@@ -5,27 +5,44 @@
 import fileinput
 import SPEFile
 import os
+import sys
+
 energy_list = [351.93, 583.19, 609.31, 911.20, 1460.82, 1764.49,
                2614.51]
 cal_headers = [351.93, 583.19, 609.31, 911.20, 1460.82, 1764.49,
                2614.51, 'Output']
 
-
 def acquire_files():
     """
     acquire_files gathers all the .Spe file in the current file directory and
     returns a list containing all .Spe files.
-    Files is a list of .spe files.
     """
     sample_measurements = []
+    sample_names = []
     dir_path = os.getcwd()
+    skip_file = ''
     for file in os.listdir(dir_path):
         if file.lower().endswith(".spe"):
-            if file == "USS_Independence_Background.Spe":
+            # Ignore the background and reference spectra
+            if file == skip_file:
+                pass
+            elif file == "USS_Independence_Background.Spe":
+                pass
+            elif file == "UCB018_Soil_Sample010_2.Spe":
                 pass
             else:
+                if '_recal' in file:
+                    skip_file = file
+                    skip_file.replace('_recal.Spe', '.Spe')
+                else:
+                    pass
+
                 sample_measurements.append(file)
-    return sample_measurements
+                name = os.path.splitext(file)[0].replace("_", " ")
+                sample_names.append(str(name))
+                sample_names.sort()
```

Comment:
Two issues I see here. One, it looks like only `sample_names` is sorted but not `sample_measurements`. Also, it's sorting in every iteration of the `for` loop, when it really only needs to be sorted once.

I suggest constructing the `sample_measurements` list, then after the `for` loop sort `sample_measurements`, then construct `sample_names = [os.path.splitext(f)[0].replace("_", " ") for f in sample_measurements]`. Then you know that `sample_names` matches `sample_measurements`.